api/src/main/java/com/epam/pipeline/manager/pipeline/PipelineConfigurationManager.java

```diff
@@ -173,14 +173,14 @@ public PipelineConfiguration mergeParameters(PipelineStart runVO, PipelineConfig
             configuration.setInstanceDisk(defaultConfig.getInstanceDisk());
         }
         if (runVO.getInstanceType() != null) {
-            configuration.setInstanceType(String.valueOf(runVO.getInstanceType()));
+            configuration.setInstanceType(runVO.getInstanceType());
         } else {
             configuration.setInstanceType(defaultConfig.getInstanceType());
         }
-        if (runVO.getTimeout() != null) {
```

Comment:
I think we need to keep 'timeout' field as well

Reply:
Oops. Fixed.